extensions/wikia/ContentReview/maintenance/setLastRevisionsAsReviewed.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+$dir = dirname( __FILE__ ) . "/../../../../";
+require_once( $dir . 'maintenance/Maintenance.php' );
+
+use Wikia\ContentReview\Models;
```

Comment:
I'm afraid that's to early to make use of `use` :)

MW autoloader is available in `execute` method.